src/main/scala/ru/otus/sc/App.scala

```diff
@@ -1,22 +1,123 @@
+import ru.otus.sc.countdown.model.{CountdownRequest, CountdownResponse}
+import ru.otus.sc.countdown.service.CountdownService
+import ru.otus.sc.countdown.service.impl.CountdownServiceImpl
+import ru.otus.sc.counter.dao.impl.CounterDaoImpl
+import ru.otus.sc.counter.model.{CounterRequest, CounterResponse}
+import ru.otus.sc.counter.service.CounterService
+import ru.otus.sc.counter.service.impl.CounterServiceImpl
+import ru.otus.sc.echo.dao.impl.EchoDaoImpl
+import ru.otus.sc.echo.model.{EchoRequest, EchoResponse}
+import ru.otus.sc.echo.service.EchoService
+import ru.otus.sc.echo.service.impl.EchoServiceImpl
 import ru.otus.sc.greet.dao.impl.GreetingDaoImpl
 import ru.otus.sc.greet.model.{GreetRequest, GreetResponse}
 import ru.otus.sc.greet.service.GreetingService
 import ru.otus.sc.greet.service.impl.GreetingServiceImpl
+import ru.otus.sc.reverse.dao.impl.ReverseDaoImpl
+import ru.otus.sc.reverse.model.{ReverseRequest, ReverseResponse}
+import ru.otus.sc.reverse.service.ReverseService
+import ru.otus.sc.reverse.service.impl.ReverseServiceImpl
+import ru.otus.sc.storage.dao.impl.StorageDaoImpl
+import ru.otus.sc.storage.model.{StorageRequest, StorageResponse}
+import ru.otus.sc.storage.service.StorageService
+import ru.otus.sc.storage.service.impl.StorageServiceImpl
+import ru.otus.sc.sum.dao.impl.SumDaoImpl
+import ru.otus.sc.sum.model.{SumRequest, SumResponse}
+import ru.otus.sc.sum.service.SumService
+import ru.otus.sc.sum.service.impl.SumServiceImpl
+
+// Helper class which aggregates service entries to single point
+case class Config(
+    countdowning: CountdownService,
+    counting: CounterService,
+    echoing: EchoService,
+    getting: StorageService,
+    greeting: GreetingService,
+    reversing: ReverseService,
+    summing: SumService
+)
+
+object Config {
+  def apply(): Config = {
+    val CountdownDao     = new CountdownDaoImpl
+    val countdownService = new CountdownServiceImpl(CountdownDao)
+    val CounterDao       = new CounterDaoImpl
+    val counterService   = new CounterServiceImpl(CounterDao)
+    val echoDao          = new EchoDaoImpl
+    val echoService      = new EchoServiceImpl(echoDao)
+    val greetingDao      = new GreetingDaoImpl
+    val greetingService  = new GreetingServiceImpl(greetingDao)
+    val storageDao       = new StorageDaoImpl
+    val storageService   = new StorageServiceImpl(storageDao)
+    val reverseDao       = new ReverseDaoImpl
+    val reverseService   = new ReverseServiceImpl(reverseDao)
+    val sumDao           = new SumDaoImpl
+    val sumService       = new SumServiceImpl(sumDao)
+
+    Config(
+      countdownService,
+      counterService,
+      echoService,
+      storageService,
+      greetingService,
+      reverseService,
+      sumService
+    )
+  }
+}
 
 trait App {
+  // provide counter started from 1 with auto-increase value
+  // every call increases counter value by 1
+  // counter can be reset to initial value with `clear` flag
```

Comment:
```suggestion
   /**
    * Provide counter started from 1 with auto-increase value
    * every call increases counter value by 1
    * counter can be reset to initial value with `clear` flag
    * @param request - ???
    * @return - ???
    */
```

То же самое замечание по комментариям.

Reply:
принято